apps/zhp-accounts-backend/Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: help
+
+help:
+	@echo "Usage: make [command]"
+	@echo "    Available commands:"
+	@echo "    - help    Show this help message"
+	@echo "    - venv    Activate Python virtual environment"
+	@echo "    - install    Install Python dependencies"
+	@echo "    - start:local    Run the functions locally (requires azure-functions-core-tools)"
+
+venv:
+	. ./venv/Scripts/activate
+
+install:
+	pip install -r requirements.txt
+
+start:local:
+	func start
```

Comment:
Pytanie, czy nie chcemy mieć już tylko taskfile, zamiast makefile

Reply:
A w sumie najpierw zrobiłem makefile, potem taskfile i chciałem usunąć makefile, ale zapomniałem. To usunie się jakoś w międzyczasie